packages/amplify-app/src/index.js

```diff
@@ -423,8 +423,10 @@ async function createIosHelperFiles() {
     fs.writeFileSync(amplifyConfigFile, configJsonStr);
   }
 
+  console.log('Checking for existing amplify project...');
   if (fs.existsSync(path.join(amplifyDir, 'backend'))) {
     await addAmplifyFiles();
+    console.log('Generating Amplify configuration files');
```

Comment:
Since this will print after `await addAmplifyFiles();`, I suggest changing the text, change the order of the statement, or use an ora spinner:

```js
await addAmplifyFiles();
console.log('Generated Amplify configuration files');
```

```js
console.log('Generating Amplify configuration files');
await addAmplifyFiles();
```

```js
// Something like...
spinner.start('Generating Amplify configuration files...');
await addAmplifyFiles();
spinner.succeed();
```